Find a functionally equivalent code:android.view.ViewGroup.handleFocusGainInternal:COMMENT
Method Modifier: 
Comment:/**
 * {@inheritDoc}
 */

@@ -1,6 +1,6 @@
 {
     if (mFocused != null) {
-        mFocused.unFocus();
+        mFocused.unFocus(this);
         mFocused = null;
     }
     super.handleFocusGainInternal(direction, previouslyFocusedRect);

